cstruct-async.opam

```diff
@@ -14,6 +14,7 @@ build: [
 ]
 depends: [
   "jbuilder" {build & >="1.0+beta7"}
+  "async"
```

Comment:
You can still access Deferred.it by opening `Async_kernel`. Note that that is where Deferred.t it defined. Async only re-exports it.